prov/efa/src/rxr/rxr_read.c

```diff
@@ -137,10 +177,10 @@ struct rxr_read_entry *rxr_read_alloc_entry(struct rxr_ep *ep, int entry_type, v
 		read_entry->addr = tx_entry->addr;
 
 		read_entry->iov_count = tx_entry->iov_count;
-		read_entry->iov = tx_entry->iov;
+		memcpy(read_entry->iov, tx_entry->iov, tx_entry->iov_count * sizeof(struct iovec)) ;
```

Comment:
Is this (and the next three memcpy's added) related to this change? Or is this a separate bug?

Reply:
It is not a bug, but a related change. Previously, we use pointer for ```iov``` and ```rma_iov```. They are just pointing to ```iov``` and ```rma_iov``` in tx_entry and rx_entry.

With this change, we have to construct ```iov``` and ```rma_iov``` on the fly, so I change them from pointer to array.

I added a separate PR for this change in new revision.